File: Java/bowling_game_kata/src/main/java/model/Lane.java
package model;

public class Lane {
    private int currentPinsStanding;


    public Lane() {
        this.currentPinsStanding = 10;
    }

    public int getPinsStanding() {
        return currentPinsStanding;
    }

    public void loadARound() {
        this.currentPinsStanding = 10;
    }
}

File: C#/Bowling_Game/Model/Lane.cs
using System;

namespace Bowling_Game.Model {
    public class Lane {
        private int pinsStanding;

        public Lane() {
            pinsStanding = 10;
        }

        public int GetPinsStanding(){
            return pinsStanding;
        }

        public void Remove(int pinsKnockedDown) {
            if (PinsCanBeKnockedDown(pinsKnockedDown)) {
                pinsStanding -= pinsKnockedDown;
            }
        }

        private bool PinsCanBeKnockedDown(int pinsKnockedDown) {
            return pinsKnockedDown > -1 && pinsKnockedDown < 11;
        }
    }
}
